src/utils/firebaseConfig.ts

```diff
@@ -0,0 +1,22 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getFirestore} from 'firebase/firestore';
+
+
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web firebaseConfig's Firebase configuration
+const firebaseConfigure = {
+    apiKey: process.env.REACT_APP_FIREBASE_APIKEY,
```

Comment:
Use a .env file to store sensitive information

Reply:
Already using a separate .env file to store sensitive information and access them via process.env ?